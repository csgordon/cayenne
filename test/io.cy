module example$io =
#include Prelude
    do Monad_IO
        (args :: List String) <- getArgs
        putStr "arg 1 is '"
        putStr (head args)
        putStr "'\n"
