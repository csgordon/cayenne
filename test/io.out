arg 1 is 'hello'
